.cljfmt/indents.clj

```diff
@@ -1,2 +1,33 @@
-{metabase.api.collection/foo [[:inner 0]]
- toucan.db/insert!           [[:block 1]]}
+;; https://github.com/weavejester/cljfmt/blob/master/cljfmt/resources/cljfmt/indents/clojure.clj
+{auto-parse                                 [[:inner 0]]
```

Comment:
Could you run Clojure LSP and have it dump out the indents file it generates based on the `:style/indent` metadata?

Reply:
I've asked around about this and IDK if it's a feature of lsp. Do you know how to do it? I _did_ run lsp on our protocol.clj file and it also does weird formatting. Maybe we can exclude that file from formatting until we find a more permanent solution.